spec/services/subscriptions/terminate_service_spec.rb

```diff
@@ -33,6 +33,28 @@
         expect(result.error).to eq('not_found')
       end
     end
+
+    context 'when pending next subscription' do
+      let(:subscription) { create(:subscription) }
+      let(:next_subscription) do
+        create(
+          :subscription,
+          previous_subscription: subscription,
+          status: :pending,
+        )
+      end
+
+      before { next_subscription }
+
+      it 'canceled the next subscription' do
+        result = terminate_service.terminate(subscription.id)
+
+        aggregate_failures do
+          expect(result).to be_success
+          expect(next_subscription.reload).to be_canceled
```

Comment:
What about writing it like this?

```ruby
expect { terminate_service.terminate(subscription.id) }
  .to change { next_subscription.reload.status }.from('pending').to('canceled')
```

Reply:
@rsempe it good but to be honest, I find it a bit harder to read